spec/features/consumer_search_on_results_list_spec.rb

```diff
@@ -0,0 +1,84 @@
+RSpec.feature 'Consumer views advice filters on search results page' do
+  let(:landing_page) { LandingPage.new }
+  let(:results_page) { ResultsPage.new }
+
+  scenario 'Consumer views current filters applied to their search' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_am_on_the_rad_landing_page
+      when_i_search_with_a_reading_postcode
+      then_i_am_on_search_results_page
+      and_advice_filters_are_shown
+      and_previously_selected_filters_are_selected_by_default
+    end
+  end
+
+  scenario 'Consumer views updated search results' do
+    with_elastic_search! do
+      given_some_firms_were_indexed
+      and_i_am_on_search_results_page_with_some_filters_selected
```

Comment:
Perhaps `and_i_have_made_a_search_from_the_landing_page` or similar? With that, you can walk through the steps required to setup the state for the search page, assert for the filters that were previously selected and then continue as is from there.